conditions.go

```diff
@@ -79,6 +79,13 @@ func GenerateConditions(ctx Context, md []ConditionMetadata) []models.Condition
 		}
 	}
 
+	afibChance := rand.Intn(100)
+	// 3% Chance of AFib in the general population according to Chris [citation needed]
```

Comment:
If considering the birth date is fairly trivial (i.e., can be implemented in less than an hour), let's do that.  Otherwise, let's bump it to 9% and add a TODO.